package.json

```diff
@@ -12,7 +12,9 @@
     "serve": "docusaurus serve",
     "write-translations": "docusaurus write-translations",
     "write-heading-ids": "docusaurus write-heading-ids",
-    "format": "npx prettier . --write"
+    "format": "npx prettier . --write",
+    "lint": "markdownlint-cli2 **/*.md #node_modules",
+    "lint:fix": "markdownlint-cli2 **/*.md **/*.mdx #node_modules --fix"
```

Comment:
Neat, do you suggest running `npm run lint` and `npm run lint:fix` prior to making any new PR's just so a cleanup like this doesn't have to happen often?

Reply:
Yes, exactly 😄 . I should have said that clearly.

@roberthunterjr please also note the new `npm` commands here. 

Also, for both of you: `npm run format` uses Prettier, as you can see. There is very little control over Markdown settings in Prettier. Therefore, I added `*.md` to the `.prettierignore` file. But, `npm run format` will still work on `.mdx`, `.jsx`, `.css`, `.yml`, and `json`. On `mdx` files, it may be good to first run the `format` command to format the react part of the file, then run `lint:fix` next which will fix up some "damage" done by Prettier.

Reply:
@stephenfuqua could you update DEVELOPER.md to add instruction to run the linter before PR?